scripts/build/termux_step_install_license.sh

```diff
@@ -56,8 +56,12 @@ termux_step_install_license() {
 					# We only want to include the license files from the source files once
 					if (( ! FROM_SOURCES )); then
 						local FILE
+						local EXTRA_LICENSE_FILES=()
+						if [ "$LICENSE" = 'MIT' ]; then
```

Comment:
I've been standardizing this script to use `[[` Bash test `]]`
```suggestion
						if [[ "$LICENSE" == 'MIT' ]]; then
```

Reply:
Thanks, committed this.